.env

```diff
@@ -1,5 +1,5 @@
 REACT_APP_STORE_PROVIDER=https://develop.kleros.in
 REACT_APP_PROD_ARBITRATOR_ADDRESS=0x6183694b0f853dff99cde9ad322973ac580b2127
-REACT_APP_DEV_ARBITRATOR_ADDRESS=0x7a5a9d8235a7bd3cd75adc8f1089c51da113b2cc
+REACT_APP_DEV_ARBITRATOR_ADDRESS=0x6183694b0f853dff99cde9ad322973ac580b2127
```

Comment:
You want the other address `0x7a5a9d8235a7bd3cd75adc8f1089c51da113b2cc` as DEV. Can you switch it back?